core/commands/generate_product_repositories_command.rb

```diff
@@ -146,7 +157,9 @@ def load_configuration_file
 
   def determine_products_to_parse
     if @env.nodeProduct
+      @env.nodeProduct = get_product_by_alias(@env.nodeProduct)
```

Comment:
Do not modify properties of arguments that are not created by the class itself. The environment variable is a global one. Should be used in read-only mode